plugins/foreman_remote_execution/index.md

```diff
@@ -1,7 +1,7 @@
 ---
 layout: plugin_index
 title: Foreman Remote Execution Documentation
-versions: [0.1, 0.0]
+versions: [nightly, 0.1, 0.0]
```

Comment:
I don't follow, do you think that people would choose nightly even when they run specific version?